src/connector/src/source/filesystem/opendal_source/opendal_reader.rs

```diff
@@ -57,14 +62,15 @@ impl<Src: OpendalSource> SplitReader for OpendalReader<Src> {
         splits: Vec<OpendalFsSplit<Src>>,
         parser_config: ParserConfig,
         source_ctx: SourceContextRef,
-        _columns: Option<Vec<Column>>,
```

Comment:
Previously, the columns variable was unused, does it mean it is None?

Reply:
No, it's just not used. Here the columns is Some(`user_defined_columns`, `hidden_columns`).

I just run some test to verify
```
CREATE Table test_s3(
        id int,
        age int,
    ) WITH (
        connector = 's3',
        match_pattern = '*.parquet',
    s3.region_name = 'custom',
    s3.bucket_name = 'hummock001',
    s3.credentials.access = 'hummockadmin',
    s3.credentials.secret = 'hummockadmin',
    s3.endpoint_url = 'http://hummock001.127.0.0.1:9301',
    ) FORMAT PLAIN ENCODE PARQUET;
```
and the columns here is `Some([Column { name: "id", data_type: Int32, is_visible: true }, Column { name: "age", data_type: Int32, is_visible: true }, Column { name: "_row_id", data_type: Serial, is_visible: false }, Column { name: "_rw_s3_v2_file", data_type: Varchar, is_visible: false }, Column { name: "_rw_s3_v2_offset", data_type: Varchar, is_visible: false }])`